src/main/resources/edu/cornell/kfs/module/purap/document/datadictionary/CuVendorCreditMemoDocument.xml

```diff
@@ -20,7 +20,15 @@
    
     <!-- workflow attributes -->
     <bean id="VendorCreditMemoDocument-workflowAttributes" parent="VendorCreditMemoDocument-workflowAttributes-parentBean">
-    	<property name="searchingTypeDefinitions">
+    	<property name="routingTypeDefinitions">
+            <map>

```

Comment:
You may want to specify the `merge="true"` attribute here, so that you only need to include the CU-specific routing definitions below (unless there's something else overriding things that requires explicitly specifying the base entries below).